### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by pavel-kirienko**
See commits. The version number is to be incremented after the other pending PRs are merged. (I wonder if it could be somehow automated, perhaps?)

### CODE REVIEW

**Review by thirtytwobits — APPROVED** [review]

**Comment by pavel-kirienko** [comment]
@thirtytwobits would you like me to merge it now, or should I wait until the other one is merged, or do you prefer to manage it yourself?

**Comment by thirtytwobits** [comment]
Please rebase, bump the version.py to 1.0 and resubmit. We'll use this to publish the 1.0 release.

**Pull request merged**
